README.md

```diff
@@ -45,10 +45,40 @@ Run the following commands from the root of the project directory:
 
 Open your forked repo in your browser and then raise a PR to the `master` branch of this repository!
 
+## Usage
+Once the executable file is created, the program can be executed as:
 
+### Without Flag Options
+This will by default show all the parameters of the System. It will provide all the parameters of CPU, Memory and Network Status.
+```console
+./termiboard
+```
+Output:   
+![Termiboard Output SS](images/Termiboard.PNG)
+### With Flag Options
+If the flag options are passed, then the specific parameters of CPU, Memory and Network Status will be displayed.
+```console
+./termiboard --cpu-usage
+```
+Output:       
+![CPU Info Output SS](images/CpuInfo.PNG)
+#### All Available Options
+```console
+./termiboard --help
+```
+| Flag      | Function                                        |
+|-----------|-------------------------------------------------|
+| all       | Show all stats                                  |
+| cpu-info  | Show CPU information                            |
+| cpu-usage | Show CPU usage                                  |
+| disk      | Show disk usage                                 |
+| local-ip  | Show local IP address                           |
+| public-ip | Show public IP address                          |
+| ram       | Show RAM usage                                  |
+| top5-ram  | Show top 5 process that consume the most memory |
```

Comment:
i feel you can append the prefix -- to all the rows in the flag coulmn

Reply:
Yes, okay